analyze/aps-dnc.c

```diff
@@ -1575,6 +1586,9 @@ static void *get_edges(void *vaug_graph, void *node) {
 	    sink.attr = (Declaration)m;
 	    record_condition_dependencies(&sink,cond,aug_graph);
 
+	    // Traverse the case_expr and add edge from things inside the case_expr to Match inside Case 
+	    record_expression_dependencies(&sink, cond, control_dependency, NO_MODIFIER, some_case_stmt_expr(decl), aug_graph);
+
```

Comment:
Isn't this done already on lines 1579-1580? (I thought match_test was the case expression?  )